src/preload.js

```diff
@@ -14,7 +14,12 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-const { ipcRenderer } = require('electron');
+const { ipcRenderer, desktopCapturer } = require('electron');
 
 // expose ipcRenderer to the renderer process
 window.ipcRenderer = ipcRenderer;
+
+// expose desktopCapturer to the render process to make screen-sharing work
+process.once('loaded', () => {
+  global.desktopCapturer = desktopCapturer;
```

Comment:
Should the ipc renderer be exposed here too? Or is there a reason one is done on the loaded event and one not?

Reply:
The `ipcRenderer` came off as weird to me as well but I am not very knowledgeable when it comes to Electron, so I am not really sure.

You're definitely right about the loaded event. That doesn't seem to be necessary. It's fixed now.